packages/@uppy/tus/src/index.js

```diff
@@ -178,6 +182,9 @@ module.exports = class Tus extends BasePlugin {
 
     // Create a new tus upload
     return new Promise((resolve, reject) => {
+      let queuedRequest
+      let queueRequest
```

Comment:
It's very easy to misuse one for the other. Maybe `queuedRequest` and `qRequest` ?